sdk/cloudmachine/Azure.CloudMachine.OpenAI/src/OpenAIFeature.cs

```diff
@@ -23,7 +24,7 @@ protected override ProvisionableResource EmitResources(ProjectInfrastructure clo
         return cognitiveServices;
     }
 
-    protected override void EmitConnections(ConnectionCollection connections, string cmId)
+    protected override void EmitConnections(ICollection<ClientConnection> connections, string cmId)
```

Comment:
Why are we changing `ConnectionCollection` to `ICollection<ClientConnection`?

Reply:
Maybe. But Don't do anything for now in your code. i.e. you can pass ConnectionCollection to this parameter.